CMakeLists.txt

```diff
@@ -495,6 +495,76 @@ while(library_variants_copy)
             -DLIBCXX_ENABLE_WIDE_CHARACTERS=OFF
             -DLIBUNWIND_ENABLE_THREADS=OFF
         )
+
+        #picolibc tests
+        string(REPLACE " " ";" qemu_params "${qemu_params}")
+        add_custom_target(check-picolibc_${variant})
+        set(qemu_command "qemu-system-arm")
```

Comment:
This can be changed to `set(qemu_command "qemu-system-${cpu_family}")` and the following `if` statement can be removed.